chapters.md

```diff
@@ -17,7 +17,62 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 
 Three `Edition Flags` are defined to describe the bevahior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
 
-The `EditionFlagHidden` Flag behaves similarly to the `ChapterFlagHidden` Flag: if `EditionFlagHidden` is set to `true` then its `Child ChapterAtoms Elements` MUST also be interpretted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden` flags. If the `EditionFlagHidden` is toggled by a `Control Track` to `false` then the `ChapterFlagHidden` Flags of the `Child ChapterAtoms Elements` SHALL determine if the `ChapterAtom` is hidden or not.
+
+#### `EditionFlagHidden`
+
+The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`: if `EditionFlagHidden` is set to `true`, its `Child ChapterAtoms Elements` MUST also be interpreted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden Flags`. If `EditionFlagHidden` is toggled by a `Control Track` to `false`, the `ChapterFlagHidden Flags` of the `Child ChapterAtoms Elements` SHALL determine whether the `ChapterAtom` is hidden or not.
```

Comment:
"`ChapterFlagHidden Flags`" has the wrong backtick, if should be "`ChapterFlagHidden` Flags"